mobile/src/Leaderboard.js

```diff
@@ -37,6 +37,17 @@ export default class Leaderboard extends PureComponent {
     )
   }
 }
+
+renderFirstPosition = (p, i) => {
```

Comment:
This function is almost entirely duplicated from the code in line 29.  I think the only difference is the avatar size?  Just vary that without duplicating everything.